bindings/wasm/manifold.d.ts

```diff
@@ -42,8 +42,8 @@ interface Vector<T> {
   set(idx: number, value: T): void;
   size(): number;
 }
-class Vector_vec3 implements Vector<SerializedVec3>{};
-class Vector_vec4 implements Vector<SerializedVec4>{};
+type Vector_vec3 = Vector<SerializedVec3>;
+type Vector_vec4 = Vector<SerializedVec4>;
```

Comment:
@rafern, I noticed TS didn't like a few of your changes here, so I updated them to I think what you were intending? Mind reviewing this file?

Reply:
Yup, that is what I intended. I never got an error on my end because I've been using esbuild instead of tsc, need to fix that on monday